content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,138 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the Git user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+3. Don't forget to install DVC and any other requirements in your environment!
+
+   ```bash
+   pip install dvc dvclive
+   ```
+
+## Notebooks
+
+After completing the setup, you can work with DVC in SageMaker notebooks like
+you would in any other environment. Take a look at DVC [experiments] for how to
+get started with DVC in notebooks (if you have setup [code-server] on SageMaker,
+you can also install the [DVC extension for VS Code]).
+
+If you would like to see live experiment updates in [DVC Studio], get your
+[token] and save it in your [dvc config] or `DVC_STUDIO_TOKEN` environment
+variable. For example, to set it globally for all of a user's projects:
+
+```cli
+$ dvc config --global studio.token ***
+```
+
+While the experiment runs, you will see live updates like this in DVC Studio:
+
+![Notebook](/img/sagemaker-notebook.gif)
+
+## Pipelines
+
+You can run SageMaker jobs in DVC [pipelines] or convert existing SageMaker
+pipelines into DVC pipelines. This combines the benefits of SageMaker jobs, like
+running each stage on its own EC2 instance and enabling other data
+[input modes](https://docs.aws.amazon.com/sagemaker/latest/dg/model-access-training-data.html),
+with the benefits of DVC pipelines, like skipping unchanged stages and tracking
+the inputs and outputs of each run. SageMaker expects all inputs and outputs to
+be stored in S3, so the easiest way to integrate with DVC is to use [external
+dependencies and outputs].
+
+### Example: XGBoost pipeline
+
+For an example, see https://github.com/iterative/sagemaker-pipeline, which
+adapts an existing SageMaker tutorial from a notebook into a DVC pipeline. The
+first stage downloads the data and tracks the output so that it doesn't have to
+be re-downloaded on each run. We parametrize the `bucket` and `prefix` of the
+destination into a separate `params.yaml` file so they can be modified easily.
+The DVC pipeline stage is defined in `dvc.yaml` like this:
+
+```yaml
+prepare:
+  cmd:
+    - rm -f bank-additional.zip
+    - wget
+      https://sagemaker-sample-data-us-west-2.s3-us-west-2.amazonaws.com/autopilot/direct_marketing/bank-additional.zip
```

Comment:
```suggestion
      https://sagemaker-sample-data-us-west-2.s3-us-west-2.amazonaws.com/autopilot/direct_marketing/bank-additional.zip -O bank-additional.zip
```
This will override `bank-additional.zip` so you don't have to `rm` it in a separate command before